bundles/org.openhab.binding.solax/README.md

```diff
@@ -0,0 +1,210 @@
+# Solax Binding
+
+This is a OpenHAB binding for a Solax Wi-fi modules that have the option to be connected directly via HTTP (firmware version 3.x+).
+Please note that earlier firmware releases do not support direct connection.
+
+Currently the Solax cloud services provide an update on every 5 minutes (sometimes even more) and they also happen to be down sometimes, which makes it hard to automate the decision making by OpenHAB if we have more complex rules.
+
+For example: I want to charge my car if the battery of my inverter is 90%+ and my overall consumption is only from the solar power.
+The binding retrieves structured data from the wifi module, parses it and pushes it into the inverter thing where each channel represents a specific information (inverter output power, voltage, PV1 power, etc)
+
+
+Hostname/IP address of the wi-fi module (can be any IP address from internal DHCP wi-fi network) [String]
+The serial number of the Wi-fi module is the default password (not sure if it can be changed) [String]
+Time interval to poll the data from the module [number in seconds]
+Please give it a try and provide me a feedback if it works for you. I will update the releases as soon as I implement some of the backlog.
```

Comment:
This documentation is probably not the best place for this comment. Perhaps some kind of announcement in the forum would be better?